extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/KafkaSamplerSpecTest.java

```diff
@@ -139,6 +123,7 @@ public void testSample()
         null,
         new KafkaSupervisorIOConfig(
             TOPIC,
+            new JsonInputFormat(new JSONPathSpec(true, ImmutableList.of()), ImmutableMap.of()),
```

Comment:
nit: this, and other similar instantiations could just use `JSONPathSpec.DEFAULT` I think

Reply:
Thanks, fixed.